Makefile

```diff
@@ -22,6 +24,8 @@ lint: lint-python
 lint-python:
 	pipenv run ./scripts/run_lint_python.sh
 
+test-lint-python: lint-python
```

Comment:
Think this one is just running lint at the moment and not test? Also minor but as we always run lint before tests I would rename these to be lint-test-python etc

Reply:
Yes, that is correct, I've made those changes.